tools/ab_test.py

```diff
@@ -158,7 +158,8 @@ def load_data_series(report_path: Path, revision: str = None, *, reemit: bool =
 
 def collect_data(binary_dir: Path, tests: list[str]):
     """Executes the specified test using the provided firecracker binaries"""
-    revision = binary_dir.name
+    # ../build/main/build/cargo_target/x86_64-unknown-linux-musl/release
```

Comment:
leftover?

Reply:
Just an example of how one of those looks like

Reply:
oh ok. Then, nit, but could you just amend the comment to say that?

Reply:
Done!

Reply:
thanks!